app/components/trainee_name/view.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if display_text %>
+<span class="govuk-caption-l"><%= display_text %></span>
```

Comment:
FYI you can use `render?` in the component and have it conditionally instantiate/render based on this value rather than adding it here (we have an example of it in the other components)